examples/kitchen-sink.yaml

```diff
@@ -262,6 +262,8 @@ meter_provider:
           included:
             - key1
             - key2
+          excluded:
```

Comment:
Can you add a comment for `excluded`, and describe the priority between `included` and `excluded`?

Reply:
Done!